src/lib/phdi-building-blocks/phdi_building_blocks/fhir.py

```diff
@@ -204,10 +206,19 @@ def _compose_export_url(
 def __export_from_fhir_server_poll_call(
     poll_url: str, access_token: str
 ) -> Union[requests.Response, None]:
-    """Poll to see if the export files are ready.  If export is still in progress,
-    and we should return null so polling continues.  If the response is 200, then
-    the export files are ready, and we return the HTTP response.  Any other status
-    either indicates an error or unexpected condition.  In this case raise an error.
+    """
+    Helper method to see if the export files are ready based on received status
+    code. If export is still in progress, then we should return null so polling
+    continues. If the response is 200, then the export files are ready, and we
+    return the HTTP response. Any other status either indicates an error or
+    unexpected condition. In this case raise an error.
+    :param str poll_url: The endpoint the FHIR server gave us to query for if
+    our files are ready
+    :param str access_token: The access token we use to authenticate with the
+    FHIR server
+    :return: None if the files didn't get processed in time, OR a response with
+    the relevant status codes and exported records if the files did get processed
+    :rtype: Either None, or a Response from Requests
```

Comment:
Looks like whatever this comment is referring to no longer exists in the code. It _might_ be referring to the helper function on lines 182-214 that checks polling status, which is why I'm going to leave it unresolved even though github marked it as outdated. Given what Spencer has said about this lining up with the `polling` module's behavior, I would argue we should not write another helper function to standardize the return type and should instead leave the code on 182-214 as is. The user should never be calling this function directly since it's only used implicitly by the actual `export_from_fhir_server_poll` on line 236, which means they shouldn't be confused about the return types. Conversely, adding yet another function to this function chain would seem to me to make working with the `polling` module more difficult, and create the same problem you had with the `geo` module, Bryan, where there were too many helper functions getting called in a chain.